event.py

```diff
@@ -0,0 +1,271 @@
+import discord
+import datetime
+from discord.ext import commands
+from discord import app_commands
+import time
+import asyncio
+import difflib
+import main
+
+bot = main.bot
+
+class BotEvents(commands.Cog):
```

Comment:
Please consider adding a class description
```suggestion
class BotEvents(commands.Cog):
    """BotEventsClass
        Its function will be registered when bot will start
    """
```